documentation/release_notes.markdown

```diff
@@ -10,6 +10,16 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
+## Puppet Server 6.12.1
+
+Released 14 July 2020
+
+### Enhancements
+
+- Jolokia will no longer log at debug level by default, which avoids large stacktraces for missing metrics. In order to re-enable debug output, set `debug` to `true` in `metrics.conf`, in addition to configuring the logging to `debug` in `logback.xml`. [TK-488](https://tickets.puppetlabs.com/browse/TK-488)
```

Comment:
The comma before "in addition to" can come out.

Reply:
ooh, yeah, I'm actually just gonna change it to "and configure the logging"